src/img/icons/upvote.svg

```diff
@@ -0,0 +1 @@
+<svg width="10" height="6" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0)"><path fill-rule="evenodd" clip-rule="evenodd" d="M9.65 5.06c.1-.1.15-.21.1-.26L4.97.01c-.017-.017-.048-.02-.086-.01-.035-.005-.065 0-.084.02l-.083.083A.544.544 0 0 0 4.71.11l-.006.006L.02 4.81c-.04.04 0 .16.1.26l.35.34c.1.1.21.15.26.1l4.15-4.16 4.16 4.16c.04.04.16 0 .26-.1l.35-.35z" fill="#3C444D" fill-opacity=".7"/></g><defs><clipPath id="clip0"><path transform="rotate(90 9.77 0)" fill="#fff" d="M9.77 0h5.53v9.77H9.77z"/></clipPath></defs></svg>
```

Comment:
This could be further optimized and made the standard 30x30 size, but It's good enough for now.